golang/outbound.go

```diff
@@ -56,12 +56,17 @@ func (c *Connection) beginCall(ctx context.Context, serviceName string) (*Outbou
 		return nil, err
 	}
 
+	headers := callHeaders{
+		CallerName: c.localPeerInfo.ProcessName,
```

Comment:
Every service has a name.

Cn, realtime-API, mobile-driver-client, mobile-rider-client

In node we will make this a mandatory option in the tchannel constructor.